assets/app/view/game/player.rb

```diff
@@ -101,6 +101,13 @@ def render_info
               h('td.right', @game.format_currency(@player.cash - committed)),
             ]),
           ]) if committed.positive?
+
+          trs.concat([
+             h(:tr, [
+               h(:td, 'Bidding tokens'),
+               h('td.right', "#{@game.active_step.bidding_tokens(@player)} / #{@game.bidding_token_per_player}"),
```

Comment:
Using round state would be better than active_step for managing bidding tokens.

Reply:
Still would like to see `@round.bidding_tokens` using the `round_state` method. Ping me if you aren't sure what I mean.